sampledb/sample_schema.json

```diff
@@ -12,14 +12,19 @@
             "default" : "air"},
         "sample_maker": {"type" : "string"},
         "quantity": {"type" : "string"},
-        "disposition": {"type": "string"},
+        "disposition": {"type": "string",
+            "default" : "Returned to Home Institution",
+            "enum" : ["Returned to Home Institution",
+                      "Leave at BNL",
+                      "Dispose at BNL"]},
         "transport": {"type": "string",
             "default" : "Automobile",
-            "enum" : ["Automobile"]},
+            "enum" : ["Automobile",
+                      "Train",
+                      "FedEx",
```

Comment:
Maybe just delivery service? That way we don't need to enum every postage/package service.

Reply:
Sure. Does the SAF list multiple shipping services?

Reply:
I'm not certain, @dragonyanglong?